docs/HOW_TO_CREATE_AMI.md

```diff
@@ -0,0 +1,46 @@
+## Creating a Log Export Container AWS AMI 
+1. Go to the EC2 management panel and click on **Launch instances**.
+
+2. Search for "Ubuntu" and select the latest version.
+![image](https://user-images.githubusercontent.com/20745533/158170962-5beb2784-277a-45d4-b30c-94a7a1511ed1.png)
+
+3. Select your Instance Type and click on **Next: Configure Instance Details**.
+
+4. This is the most important step. You can enter custom configurations or leave it with the defaults values, but under the **Advanced Details > User data** field you need to copy and paste the content from the file [ami-cloud-init-user-data.sh](ami-cloud-init-user-data.sh). After that click on **Next: Add Storage**.
+![image](https://user-images.githubusercontent.com/20745533/158171270-ebc54d1a-86fc-44a1-8675-8cdd749ecaba.png)
+
+5. Configure your storage or leave it with the default values and click on **Next: Add Tags**.
+
+6. Add tags if you want and click on **Next: Configure Security Group**.
+
+7. Now, with the option "Create a new security group" selected, click on **Add Rule** and configure it as follows:
```

Comment:
Don't go into the details about how to configure the Security Group. Just say that we need to configure it and leave it til there.